test/ruby/Dockerfile

```diff
@@ -52,12 +52,12 @@ RUN set -ex; \
 
 RUN set -ex; \
     cd a; \
-    bundle lock
+    bundler lock
```

Comment:
Can you refresh me on `bundle` vs `bundler`?

Reply:
It's the same, renovate uses `bundler` like the package name, `bundle` is the older name.